tests/hello.c

```diff
@@ -1,5 +1,5 @@
 #include <stdio.h>
-#include "encoding.h"
+#include "riscv-pk/encoding.h"
```

Comment:
I think if you change this to `<riscv-pk/encoding.h>`, you don't need to do the PREFIX_NO_MINUS, or even add the `-I` flag to CFLAGS.

Reply:
Done